cli/spcli/actor.go

```diff
@@ -746,6 +750,10 @@ func ActorProposeChangeWorkerCmd(getActor ActorAddressGetter) *cli.Command {
 
 			ctx := lcli.ReqContext(cctx)
 
+			if cctx.NArg() != 1 {
```

Comment:
already handled at the top, let's just leave this one or you could replace it with `IncorrectNumArgsWithHint` and preserve the message